packages/cascara/src/modules/DataRadio/DataRadio.js

```diff
@@ -51,16 +60,11 @@ const DataRadio = ({
       {option.label && isLabeled && (
         <span className={styles.LabelText}>{option.label}</span>
       )}
-    </label>
+    </>
```

Comment:
label tag to wrap input

Reply:
This module is now wrapped by the new `LabeledModule` 👍